README.md

```diff
@@ -0,0 +1,76 @@
+# unity-localisation
+
+## What is it?
+A library containing classes for adding localisation support to your application.
+
+## How to use it.
+The localiser works by ingesting a set of localisation keys (the ID values for each string in your application, e.g. MainMenuTitle, MainMenuBodyText, MainMenuStartButtonText etc). These can then be used to generate a localisation class in code which declares enums containing all these keys, which the application can use to retrieve them in a type-safe way. The enum values are converted to CRC/hash values so that retrieval is optimal in terms of performance, too.
+
+For this reason, the setup includes two parts:
+* Creating a key schema, which is only used at editor-time to generate the generate the Localisation class and the key enums
+* Creating localisation tables which each represent the localised strings and language(s) they support
+
+When the application is built, the editor-side bits will not be included, leaving only the data (localisation tables) and the generated lookup class which now contains the enums of all the loc keys needed.
+
+## Editor-side
+
+### Open the Localisation editor window
+
+In Unity: Toolbar -> DUCK -> Localisation
+
+### Attach, or create, a key schema
+
+In the Localisation window, you will see a slot for assigning a key schema if you have one, and a button for generating a new one. Create or assign one - it doesn't need to be in Resources/ as it's editor-side only.
+
+Once your key schema exists, start declaring localisation keys. Sets of keys are grouped into a category for making them easier to find (e.g. UI, CharacterNames, ErrorMessages, etc). Additionally, each category may also have a Type: Text, Audio or Image. They are all strings internally, and Text will be the type you use most often - the other types are covered later in this Readme.
+
+Populate the schema with some meainingful data and keys. And make sure it's assigned in the Localisation window, as this is used to drive a lot of the rest of the localisation Editor UI. You only need one key schema per project, as it declares the keys the application knows about, not their values (yet).
+
+### Generate a Loc class
+
+This file is the 'bridge' the application will use to retrieve strings from the Localisation system. It declares all the enums the code will need to reference to do this, so for that reason, it needs to be automatically (re)generated every time you add or remove keys (or categories, or change category types, or anything else you edit in the schema). Once you're happy with your keys, assign the LocalisationConfigTemplate file in the 'Config template:' slot, and click 'Generate Loc Class'. This will autogenerate the class your application will need to reference to get strings - including the Keys enums and the Get(Enum key) function.
+
```

Comment:
` it needs to be automatically (re)generated every time you add or remove keys `

The wording here sounds like the system will automatically regenerate when you make a change? Is this so?